src/components/AppProvider/README.md

```diff
@@ -481,22 +481,27 @@ To provide access to your initialized Shopify App Bridge instance, we make it av
 import React from 'react';
 import {render} from 'react-dom';
 import * as PropTypes from 'prop-types';
-import {AppProvider} from '@shopify/polaris';
+import {
+  AppProvider,
+  _SECRET_INTERNAL_APP_BRIDGE_CONTEXT,
+} from '@shopify/polaris';
 import {Redirect} from '@shopify/app-bridge/actions';
 
 class MyApp extends React.Component {
   // This line is very important! It tells React to attach the `polaris`
```

Comment:
This comment is no longer accurate as we're referring to a specific modern context instead of a legacy 'polaris' context object that contained everything.

As this is now idiomatic Context usage I'm tempted to just remove it entirely